pre_push.py

```diff
@@ -15,36 +15,37 @@ def do_process(*args):
 
     Exit if command is not found.
     """
-    print('Running: {}'.format(' '.join(args)))
+    print("Running: {}".format(" ".join(args)))
     try:
-        check_call(args)
+        check_call(args, shell=shell)
     except CalledProcessError:
-        print('\nFailed: {}'.format(' '.join(args)))
+        print("\nFailed: {}".format(" ".join(args)))
         return False
     except Exception as exc:
-        sys.stderr.write(str(exc) + '\n')
+        sys.stderr.write(str(exc) + "\n")
         sys.exit(1)
     return True
 
 
 def main():
     """Entry point to pre_push.py."""
     success = True
-    success &= do_process('flake8', '--exclude=.eggs,docs,.tox')
-    success &= do_process('pydocstyle', 'praw')
-    success &= do_process('pylint', '--rcfile=.pylintrc', 'praw')
+    success &= do_process(["black -l 79 *.py docs praw tests"], shell=True)
+    success &= do_process(["flake8", "--exclude=.eggs,docs,.tox"])
```

Comment:
Flake8 is spamming loads of “Q000 Remove bad quotes” messages because of black’s formatting. Can we have `--ignore=Q000` be added to this command?

Reply:
What version of flake8 do you have? Do you happen to have flake8-quotes installed? Perhaps that's resulting in the issues.

> 3.7.7 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.7.2 on Darwin

That's what I'm running?

Reply:
I do happen to have flakes8-quotes installed.

> 3.7.5 (flake8_quotes: 1.0.0, mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.0) CPython 3.7.2 on Darwin